corehq/apps/enterprise/enterprise.py

```diff
@@ -383,3 +388,68 @@ def _get_individual_export_rows(self, exports, export_line_counts):
             )
 
         return rows
+
+
+class EnterpriseCommCareVersionReport(EnterpriseReport):
+    title = _('CommCare Client Version Compliance')
+    metric = _('%% of mobile workers on the latest commcare client version ')
+
+    def __init__(self, account, couch_user, **kwargs):
+        super().__init__(account, couch_user, **kwargs)
+
+    @property
+    def headers(self):
+        return [
+            _('Mobile Worker'),
+            _('Project Space'),
+            _('Latest Version Available at Submission'),
+            _('Version in Use'),
+
+        ]
+
+    def rows_for_domain(self, domain_obj):
+        rows = []
+
+        for user in get_all_user_rows(domain_obj.name,
+                                    include_web_users=False,
+                                    include_mobile_users=True,
+                                    include_inactive=False,
+                                    include_docs=True):
+            user = CouchUser.wrap_correctly(user['doc'])
+            last_submission = user.reporting_metadata.last_submission_for_user
```

Comment:
Possibly could be turned into its own function, like: `get_last_version_for_user(user)`. This could be especially important if this same logic is used in the Application Status Report.

Reply:
2e874e57c19bd581866277a544408ff90c7bc6a7